README.md

```diff
@@ -52,6 +52,8 @@ The *hostname* option sets the FQDN to the header of your emails, its optional,
 
 4. Follow Bamboo [Getting Started Guide](https://github.com/thoughtbot/bamboo#getting-started)
 
+ Optional step: You can also set `BambooSMTP.TestAdapter` as your test adapter if you expect a response that follow the format of a SMTP server raw response on your tests. 
```

Comment:
I think we should follow the list by adding a 5th element:

```
5. **Optional** Set `BambooSMTP.TestAdapter` as your test adapter:

# In your config/config.exs file
if Mix.env() == :test do
  config :my_app, MyApp.Mailer, adapter: MyApp.SMTPTestAdapter
end
```

Reply:
done!

Reply:
done!